### PR TITLE
Update eatmydata; try to activate it by default

### PR DESCRIPTION
**Pull request by anti-social**

### CODE REVIEW

File: src/builder/commands/ubuntu.rs [hunk 1]
```diff
@@ -675,57 +675,38 @@ impl Distro {
...
-                self.codename.as_ref().unwrap(), &self.config.arch);
-            if let Some(ref params) = eatmy {
-                if params.needs_universe {
-                    debug!("Add Universe for eat my data");
-                    self.enable_universe()?;
-                    self.add_universe(ctx)?;
+                if let Ok(()) = apt_get_install(ctx, &[params.package], &EatMyData::No) {
+                    self.eatmydata = params.find(ctx);
+                } else {
+                    warn!("Could not install {} package", params.package);
```
Comment (tailhook) [inline src/builder/commands/ubuntu.rs:697]: Can we also emit error here?

File: src/builder/commands/ubuntu.rs [hunk 2]
```diff
@@ -1226,37 +1209,53 @@ mod build {
...
+                        info!("Eatmydata activated: {}", p);
+                        EatMyData::Preload(p)
+                    }
+                )
+        }
+
+        pub fn _find(&self, ctx: &mut Context) -> Result<String, String> {
+            let emd_pkg_output = capture_command(
+                ctx,
+                &["/usr/bin/dpkg-query".to_string(), "-L".to_string(), self.package.to_string()],
```
Comment (tailhook) [inline src/builder/commands/ubuntu.rs:1243]: Is it fast enough to run?

File: src/builder/commands/ubuntu.rs (hunk 1 above)
Comment (anti-social) [reply src/builder/commands/ubuntu.rs:697]: I'm afraid in future ubuntu releases eatmydata package may be renamed.

File: src/builder/commands/ubuntu.rs (hunk 2 above)
Comment (anti-social) [reply src/builder/commands/ubuntu.rs:1243]: Fast enough:
```
$ time dpkg-query -L git

real    0m0.005s
user    0m0.000s
sys     0m0.005s
```

File: src/builder/commands/ubuntu.rs (hunk 1 above)
Comment (anti-social) [reply src/builder/commands/ubuntu.rs:697]: So in this way we won't break future ubuntu versions.

Comment (anti-social) [reply src/builder/commands/ubuntu.rs:697]: Oh, you mean `error!` )))

Comment (tailhook) [reply src/builder/commands/ubuntu.rs]: No I mean:
```
 match apt-get-install... {
 Err(e) => warn!("some intro {:#}", e), // << so we see actual error here
 }
```

Comment (anti-social) [reply src/builder/commands/ubuntu.rs]: I think it does not add any new information (we say clearly which package cannot be installed):
```
Could not install libeatmydata1 package. The build may be slower than usual: error running <Command "/usr/bin/apt-get" "-oDir::cache::pkgc
ache=" "-oDir::cache::srcpkgcache=" "install" "-y" "libeatmydata1-asdf"; environ[5]; chroot="/vagga/root"; work-dir="/work"> exited with code 100
```

Comment (tailhook) [reply src/builder/commands/ubuntu.rs]: Yes, but this adds information:
1. Whether the apt-get command could run (i.e. /usr/bin/apt-get is executable)
2. Whether all the respective volumes could be mounted
etc.

There are so many things that could go wrong beyond non-existence of the package. And if everything is okay, we won't get any message. So it's better to add more info if it fails.

**Comment by anti-social** [comment]
Updated.

**Comment by tailhook** [comment]
Great! Thank you.

**Pull request merged**